projects/irssi/server-fuzz.options

```diff
@@ -0,0 +1,4 @@
+[libfuzzer]
+detect_leaks = 0
+close_fd_mask = 3
```

Comment:
this will make server inefficient, can this logging be disabled at build time.

Reply:
It looks like the fuzz targets for glib itself disable stderr output using close_fd_mask and the stderr output from this irssi fuzz target is coming from glib. I assume there is a reason that the glib fuzz targets use close_fd_mask.

So I'll just disable the output from irssi that appears on stdout and change the close_fd_mask to be 2 instead of 3.